plugins/nanoleaf/NanoleafDevice.cpp

```diff
@@ -68,7 +73,19 @@ NanoleafDevice::NanoleafDevice(
  * @return true on success, false on failure
  */
 bool NanoleafDevice::StartHook() {
-  vector<uint8_t> panels;
+  string text_version = m_preferences->GetValue(VersionKey());
+
+  NanoleafNode::NanoleafVersion version = NanoleafNode::VERSION_V1;
+  if (text_version == VERSION_V1_TEXT) {
+    version = NanoleafNode::VERSION_V1;
+  } else if (text_version == VERSION_V1_TEXT) {
```

Comment:
Should be VERSION_V2_TEXT

Reply:
Done in https://github.com/peternewman/ola/pull/37